core/src/test/java/org/apache/accumulo/core/conf/AccumuloConfigurationTest.java

```diff
@@ -173,6 +173,19 @@ public String get(Property property) {
       return v;
     }
 
+    @Override
```

Comment:
Is it necessary to duplicate the function here instead of delegating to the method in AccumuloConfiguration? Anything using this method is only valid as long as both stay in sync?  (I think this applies to the other methods that that have a concrete implementation in AccumuloConfiguration

Reply:
> Is it necessary to duplicate the function here instead of delegating to the method in AccumuloConfiguration?

Yes, I believe so. The methods in AccumuloConfiguration are abstract, there is no implementation. The version of getProperties that takes the filter is also abstract, so it's duplicated in all of the subclasses as well.